.github/workflows/dapr-perf.yml

```diff
@@ -0,0 +1,157 @@
+# ------------------------------------------------------------
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+# ------------------------------------------------------------
+
+name: dapr-perf
+
+on:
+  schedule:
+    - cron: '0 */2 * * *'
+  repository_dispatch:
+
+jobs:
+  test-perf:
+    name: perf tests
+    runs-on: ubuntu-latest
+    env:
+      GOVER: 1.14.3
+      GOOS: linux
+      GOARCH: amd64
+      GOPROXY: https://proxy.golang.org
+      DAPR_REGISTRY: ${{ secrets.DOCKER_TEST_REGISTRY }}
+      DAPR_TEST_REGISTRY: ${{ secrets.DOCKER_TEST_REGISTRY }}
+      HELMVER: v3.2.1
+      DAPR_NAMESPACE: dapr-tests
+      MAX_TEST_TIMEOUT: 5400
+      DAPR_PERF_QPS: 1000
+      DAPR_PERF_CONNECTIONS: 16
+      DAPR_TEST_DURATION: 1m
+      DAPR_PAYLOAD_SIZE_KB: 1
```

Comment:
@yaron2 is this correct env vars for your environment?

Reply:
Yep